mingw-w64-bzip2-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,62 @@
+# This file was generated by gen-conda-repack-recipe
+# using the MSYS2 package epoch: Mar 02 2024 06:00:00 UTC
+#
+# Any edits will be overwritten
+#
+# https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-bzip2/PKGBUILD
+{% set pkgname = "bzip2" %}
+{% set pkgver = "1.0.8" %}
+{% set pkgrel = "3" %}
+{% set build_number = pkgrel|int * 1000 + 0 %}
+
+package:
+  name: repack_mingw-w64-bzip2
+  version: {{ pkgver }}
+
+source:
+  - url: https://repo.msys2.org/mingw/ucrt64/mingw-w64-ucrt-x86_64-bzip2-{{ pkgver }}-{{ pkgrel }}-any.pkg.tar.zst
+    folder: ucrt64-bzip2
+    sha256: 932da2c63b23e6a4448757eb36fb198a9e51121874408270afe2c91bada513c7
+
+build:
+  number: {{ build_number }}
+  binary_relocation: false
+  detect_binary_files_with_prefix: false
+  skip: True             # [not win]
+
+outputs:
+  - name: ucrt64-bzip2
+    skip: True          # [not (win and x86_64)]
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    build:
+      number: {{ build_number }}
+      missing_dso_whitelist:
+        - $RPATH/api-ms-win-crt-environment-l1-1-0.dll       # [win]
+        - $RPATH/api-ms-win-crt-filesystem-l1-1-0.dll        # [win]
+        - $RPATH/api-ms-win-crt-heap-l1-1-0.dll              # [win]
+        - $RPATH/api-ms-win-crt-math-l1-1-0.dll              # [win]
+        - $RPATH/api-ms-win-crt-private-l1-1-0.dll           # [win]
+        - $RPATH/api-ms-win-crt-runtime-l1-1-0.dll           # [win]
+        - $RPATH/api-ms-win-crt-stdio-l1-1-0.dll             # [win]
+        - $RPATH/api-ms-win-crt-string-l1-1-0.dll            # [win]
+        - $RPATH/api-ms-win-crt-time-l1-1-0.dll              # [win]
+        - $RPATH/api-ms-win-crt-utility-l1-1-0.dll           # [win]
+        - $RPATH/libbz2-1.dll                                # [win]
+    requirements:
+      build:
+      host:
+      run:
+        - msys2-mingw-w64-mutex * ucrt64
+        - ucrt64-gcc-libs                          >=13.2.0     # mingw-w64-ucrt-x86_64-gcc-libs
+    test:
+      commands:
+        - test -f $PREFIX/Library/ucrt64/bin/bunzip2.exe                         # [unix]
+        - IF NOT EXIST %PREFIX%\Library\ucrt64\bin\bunzip2.exe EXIT /B 1         # [win]
+        - bunzip2 --help                                                         # [win and x86_64]
+        - bzcat --help                                                           # [win and x86_64]
+        - bzip2 --help                                                           # [win and x86_64]
+    about:
+      home: https://sourceware.org/bzip2/
+      summary: "A high-quality data compression program (mingw-w64) [repack of MINGW-packages bzip2 for UCRT64]"
```

Comment:
@ifitchet Not a big deal but we can replicate the `summary` in the `description`.
It's not _Art_, right?
Then you can skip annoying lints by adding them at the end of `meta.yaml` files:
```
extra:
  skip-lints:
    - LINT_NAME
```

Reply:
`description` == `summary` seems common enough in real recipes so that sounds fine here. 😄 

I wasn't aware of the `skip-lints` at the time the generator was written.  We can look into that. 👍

Reply:
Doh!  I didn't do this.

Reply:
I wouldn't say it's a tragedy :)